### PR TITLE
Arash/New hero animation

### PR DESCRIPTION
**Pull request by arash-fs**
# Description

add new hero animation on the home page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/home/_hero.js [hunk 1]
```diff
@@ -1,240 +1,174 @@
...
 
     @media ${device.tabletL} {
-        justify-content: flex-start;
-        margin-top: 13.37rem;
-        margin-bottom: 4rem;
+        font-size: 4rem;
     }
 `
+const contents = [
+    'Tight spreads',
```
Comment (mustofa-binary) [inline src/pages/home/_hero.js:64]: this need to be <Localize 🙏

File: src/pages/home/_hero.js [hunk 2]
```diff
@@ -1,240 +1,174 @@
...
-}
-
-CheckMarkBullet.propTypes = {
-    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-}
-
 export const Hero = () => {
-    useLazyVideo()
+    const data = useStaticQuery(query)
+    const subtitle = localize('Trade forex, commodities, synthetic and stock indices')
```
Comment (mustofa-binary) [inline src/pages/home/_hero.js:111]: when its applying from js to jsx localize need to use <Localize /> instead

File: src/pages/home/_vertical-carousel.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+const VerticalCarousel = ({ contents }) => {
+    const [active_header, setActiveHeader] = React.useState(contents.length - 1)
+    const [transform, setTransform] = React.useState(-72)
+    const [visibility, setVisibility] = React.useState('hidden')
+
+    const startVerticalCarousel = (active_header = 5, transform = -72) => {
+        if (active_header > 0) {
+            setActiveHeader(active_header - 1)
+            setTransform(transform + 36)
```
Comment (mustofa-binary) [inline src/pages/home/_vertical-carousel.js:45]: can you help to add a comment on what are these numbers used for, or extract them to variable

File: src/pages/home/_hero.js (hunk 1 above)
Comment (arash-fs) [reply src/pages/home/_hero.js:64]: :+1:

File: src/pages/home/_vertical-carousel.js (hunk 3 above)
Comment (arash-fs) [reply src/pages/home/_vertical-carousel.js:45]: sure

File: src/pages/home/_hero.js (hunk 2 above)
Comment (arash-fs) [reply src/pages/home/_hero.js:111]: Sure

**Pull request merged**